src/components/Visualizations/Table/KDA.jsx

```diff
@@ -20,7 +20,7 @@ const KDA = ({ kills, deaths, assists, matchId = uuid.v4() }) => {
           <div style={{ width: `${(assists * 100) / kdaSum}%`, backgroundColor: palette.green }} />
         </div>
         <ReactTooltip id={`kda-${matchId}`} place="right" type="light" effect="float">
-          {`KDA: ${Number(((kills + assists) / (deaths + 1)).toFixed(2))}`}
+          {`${strings.abbr_kda}: ${Number(((kills + assists) / Math.max(deaths, 0)).toFixed(2))}`}
```

Comment:
but if player dies more than 0 times `kda` will be wrong
11 7 11 = 3.14
11 7+1 11 = 2.74